styles-media-queries.css

```
@@ -5,7 +5,7 @@
   }
 
   #copyright {
-    font-size: 0.9rem;
+    font-size: 1.1rem;
```

```suggestion
    font-size: 1.0rem;
```
Since this is in a media query, maybe we should make it slightly smaller.

---

I think my original change makes more sense.